x/contracts/wasmlanche/benches/call_contract_mem.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+use contracts::{Contract, Nft};
+use std::hint::black_box;
+use wasmlanche::Address;
+use wasmlanche_test::Builder;
+
+mod contracts;
+
+iai::main!(call_contract, call_nft_mint);
+
+fn call_contract() {
+    let builder = Builder::new("test-crate");
+    Contract::new(builder).always_true();
+}
+
+fn call_nft_mint() {
+    let builder = Builder::new("nft");
+    Nft::new(builder).mint(Address::new(black_box([2; 33])), black_box(0));
```

Comment:
🤔 I get the sense that their example doesn't make sense. Or maybe it makes sense when there's a finite number of inputs, but iter-batched and friends dynamically generate their "size" (number of times they execute), so it's impossible to optimize that. At best, it could get rid of an indirection, but that would probably be fine.